package.json

```diff
@@ -14,7 +15,8 @@
     "prepare": "npm run build && ./scripts/check-release.sh",
     "build": "npm run build:ts && npm run build:web",
     "build:ts": "tsc",
-    "build:web": "rollup -c rollup.config.js",
+    "build:web": "rollup -c rollup.config.js --bundleConfigAsCjs && npm run build:web:compress",
+    "build:web:compress": "terser build/main.min.js --compress --mangle --source-map \"content='./build/main.min.js.map'\" --output build/main.min.js",
```

Comment:
Official [@rollup/plugin-terser](https://github.com/rollup/plugins/tree/master/packages/terser) is not generating source maps for some reason. Maybe this will be fixed soon.